lib/package/plugins/checkSharedFlowBundleStructure.js

```diff
@@ -0,0 +1,252 @@
+//bundleStructure.js
```

Comment:
maybe consider the way to integrate sharedflow into existing bundleStructure plugin or create specific plugin for shared flow from scratch (but in this case need to update the docs with instructions how to skip boundleStructure plugin for sharedflow and wise versa)